packages/react-native-win32/src/Libraries/Components/TextInput/TextInput.Types.win32.ts

```diff
@@ -66,10 +66,3 @@ export type IEditingEvent = NativeSyntheticEvent<Readonly<IEditingPayload>>;
 
 // TODO: Why do I need this one
 export type IPasswordRules = string;
-
-// TODO: Can I get away without this?
```

Comment:
>? [](start = 36, length = 1)

Yes we can :)

This was only being used for a template parameter for the ObjectRef, and seemingly not being used correctly to begin with.